packages/tools/generator-component/generators/app/templates/test-utils/component-objects/f-skeleton.component.js

```diff
@@ -9,7 +9,7 @@ module.exports = class <%= name.filename %> extends Page {
         super.openComponent('<%= storybook.componentCategory.toLowerCase().replace('s','') %>', '<%= name.class %>-component');
     }
 
-    waitForTestComponent () 
+    waitForTestComponent () {
```

Comment:
Could you rename to `waitForComponent` ?

Reply:
I sure can 😄